### ISSUE TITLE
role podman destroyed my subuid & subgid

### ISSUE THREAD

**Issue by barthe** [bluebanquise/community#26]
```diff
TASK [podman : users █ Create /etc/subuid and /etc/subgid] ***********************************************************************************************************************************************************************************
Tuesday 14 September 2021  15:44:18 +0000 (0:00:00.137)       0:00:36.968 *****
--- before: /etc/subuid
+++ after: /root/.ansible/tmp/ansible-local-950273__5yru9j/tmpmz0_wx3t/subuid.j2
@@ -1,5 +1,6 @@
-aaaa:4444:65536
-bbbbb:5555:65536
-ccccc:6666:65536
+#
+# Ansible managed file, do not edit
+#
+root:100000:65535
+root:100000:65535
+guest:1000:1000

changed: [hostname] => (item=etc/subuid)
--- before: /etc/subgid
+++ after: /root/.ansible/tmp/ansible-local-950273__5yru9j/tmpc1lkollu/subgid.j2
@@ -1,5 +1,6 @@
-aaaa:4444:65536
-bbbbb:5555:65536
-ccccc:6666:65536
+#
+# Ansible managed file, do not edit
+#
+root:100000:65535
+root:100000:65535
+guest:1000:1000

```

not cool.

**Comment by strus38** [comment]
Hi
I am taking it and will propose a new PR to solve this issue.
It will be solved by adding a new variable to overwrite existing users and by default it won't deal with users.

**Comment by strus38** [comment]
I also made the same fix for flexlm.

**Comment by oxedions** [comment]
Issue linked to PR https://github.com/bluebanquise/community/pull/27

**Comment by oxedions** [comment]
Branch has been merged. I let you @strus38 close this issue as this role is managed by you directly.
